lib/administrate/search.rb

```diff
@@ -31,7 +31,7 @@ def to_s
       private
 
       def filter?(word)
-        valid_filters&.any? { |filter| word.match?(/^#{filter}:\w*$/) }
+        valid_filters&.any? { |filter| word.match?(/^#{filter}:.*$/) }
```

Comment:
I wonder: why do we even bother matching to the right of the colon? Would it be simpler to just do the following?

```suggestion
        valid_filters&.any? { |filter| word.match?(/^#{filter}:/) }
```

Reply:
Yeah, I agree 🤔 
also removes ambiguity as to what is being matched